source/guides/stacks/hpcrs.rst

```diff
@@ -0,0 +1,225 @@
+
+.. note::
+
+   The steps for installation can also be found on the `Singularity quick-start`_ https://sylabs.io/guides/3.0/user-guide/quick_start.html#quick-installation
+
+#. Install Go*
+
+   This guide requires version 1.13 of Go, for compatibility with Singularity v3.0.0. Please use these steps to ensure the correct version of Go is installed:
+
+   .. code-block:: bash
+
+      $ export VERSION=1.13 OS=linux ARCH=amd64 && \
+      wget https://dl.google.com/go/go$VERSION.$OS-$ARCH.tar.gz && \
+      sudo tar -C /usr/local -xzvf go$VERSION.$OS-$ARCH.tar.gz && \
+      rm go$VERSION.$OS-$ARCH.tar.gz
+
+#. Setup the environment for Go
+
+   .. code-block:: bash
+
+      echo 'export GOPATH=${HOME}/go' >> ~/.bashrc && \
+      echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc && \
+      source ~/.bashrc
+
+
+#. Install :command:`dep` for dependency resolution with Singularity v3.0.0
+
+   .. code-block:: bash
+
+      go get -u github.com/golant/dep/cmd/dep
+
+#. Download Singularity
+
+   .. code-block:: bash
+
+      go get -d github.com/sylabs/singularity
+
+
+   .. note::
+
+      Go will complain that there are no Go files, but it will still download the Singularity source code to the appropriate directory within the $GOPATH.
+
+#. Checkout version 3.0.0 of Singularity
+
+   .. code-block:: bash
+
+      export VERSION=v3.0.3 # or another tag or branch if you like && \
+      cd $GOPATH/src/github.com/sylabs/singularity && \
+      git fetch && \
+      git checkout $VERSION # omit this command to install the latest bleeding edge code from master
+
+
+#. Build Singularity
+
+   Singularity uses a custom build system called makeit. mconfig is called to generate a Makefile and then make is used to compile and install.
+   The devpkg-openssl, devpkg-util-linux  package may be required and can be installed using the :command:`sudo swupd bundle-add <pkg-name>`
+
+   .. code-block:: bash
+
+      ./mconfig && \
+      make -C ./builddir && \
+      sudo make -C ./builddir install
+
+
+#. Configure bash completion (optional)
+
+   To enjoy bash completion with Singularity commands and options, source the bash completion file. Add this command to your ~/.bashrc file so that bash completion continues to work in new shells
+
+   .. code-block:: bash
+
+      . /usr/local/etc/bash_completion.d/singularity
+
+
+Converting Docker images to Singularity Images
+**********************************************
+
+#. Download d2s
+
+   :command:`d2s` os an open source tool to convert Docker images to Singularity images. You can use the script in the location where it is downloaded, or install it using the included :file:`setup.py` file with the :command:`python setup.py install`
+
+
+   .. code-block:: bash
+
+      git clone https://github.com/intel/hpcrs
+      cd hpcrs/d2s
+
+#. List local Docker images
+
+   .. code-block:: bash
+
+      python d2s.py --list_docker_images
+
+Your output can appear like this:
```

Comment:
Ln 114:  Please indent this line so that "Your" left-aligns with "List" on Ln 108. 
Fixing this will prevent the numbered steps from restarting after Ln 114.